### DISCUSSION TITLE
High CPU usage when playing

### DISCUSSION THREAD

**Discussion by karniv00l** [KRTirtho/spotube#149] [General]
I'm just curious from where it's coming from and if there's a way to debug this.

This is M1 Max (app release v2.3.0):

<img width="1287" alt="Screenshot 2022-07-16 at 11 08 16" src="https://user-images.githubusercontent.com/3144291/179348497-c20cb28e-4619-4938-bcef-6e13dc3f2cca.png">

Of course, it goes to almost 0% when paused

**Comment by KRTirtho** [comment]
It's suspicious. Could be the audio plugin. But in other platforms the CPU stays below 15% (i3-3220)
